Makefile

```diff
@@ -1,6 +1,7 @@
 .PHONY: build test clean prepare update docker
 
-GO = CGO_ENABLED=0 GO111MODULE=on go
+GO=CGO_ENABLED=0 GO111MODULE=on go
+GOCGO=CGO_ENABLED=1 GO111MODULE=on go
```

Comment:
Why the change to `CGO` and why do we need both?

Reply:
Required now that the Device Services are publishing to the MessageBus which brings in the ZMQ C library

Agreed that we don't need both

Reply:
fixed and removed obsolete update target